spec/math_spec.rb

```diff
@@ -343,4 +366,23 @@
       end
     end
   end
+
+  context "#solve", focus: true do
```

Comment:
@v0dro That's not really how it works. This is an open source project. If someone raises concerns about your code, we don't merge until we feel the concerns are satisfied.

I didn't realize that `:focus => true` does what it does, or I wouldn't have let this one slip through, either. It seems like it disables other tests? What's the point in having any tests at all if one of the tests disables all the others?

This is absolutely a fix that needs to be made before I will merge.